images/pgmq-pg/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM postgres:15.4-bookworm as builder
+# syntax=docker/dockerfile:1.4
 
-ARG PGMQ_VER=0.33.1
+ARG PG_DOCKER_VERSION
+FROM postgres:${PG_DOCKER_VERSION}-bookworm as builder
```

Comment:
Need to redeclare the argument to keep using it afterwards, see https://docs.docker.com/reference/dockerfile/#understand-how-arg-and-from-interact

Reply:
Before this pull request, I was just a simple Python developer. Now, thanks to your patience and guidance, I've leveled up! Thank you!